scarlet/source.py

```diff
@@ -59,17 +61,68 @@ def get_psf_sed(sky_coord, observation, frame):
     sed = get_pixel_sed(sky_coord, observation)
 
     # approx. correct PSF width variations from SED by normalizing heights
+    if observation._diff_kernels is not None:
+        sed /= observation._diff_kernels.image.sum(axis=(-2,-1))
+
+        return sed
```

Comment:
I don't think that using `sum` in this function as opposed to `max` will give better colors, in fact I'm almost certain that it won't. In general the PSFs are normalized to sum to 1 in each band (in fact it may even be a requirement), so in most cases the sed is not affected at all by the `sum` operation.

Recall that our SED and morphology exist in the model frame before convolution with the difference kernel. This means that when we build the model in the first iteration we are convolving with the difference kernel, which will change the amplitude of the SED. That is the effect that the `max` normalization is meant to (mostly) cancel, and has significantly improved our initial colors. It also helps our initial amplitude estimates, but it isn't perfect because a pixel is also affect by the PSF from its neighboring pixels. So we could improve the amplitudes by weighting some subset of the central pixels by the PSF, which I think we did at some point and realized that it did more damage to the initial colors. With this algorithm the colors are right because it adjusts the central pixel for variations in the width of the PSF in each band.

But I do agree that if the difference kernels are already built then we could just use the _maximum_ value of the difference kernel to adjust the SED.

Reply:
Indeed, the colours are worse. 
Intuitively I don't like the max. because the central pixel is not just affected by the max pixel of the psf but all the pixels of the psf multipled by the image's pixels around the central pixel. But I surrender to the fact that in practice it works well.

Reply:
I have delayed pushing my PR because I'm testing the use of the diff_kernel here. I don't understand why, but I get better colours  when I do this :
`sed *= observation._diff_kernels.image.sum(axis=(-2,-1))`
And in the case of multi-resolution, the difference is huge! Using this leads to much better results. 
For single observations, I've only tested onn the quickstart and at e_rel = 1.e-7, it takes 193 iterations with this scheme against 237 iterations with the current implementation in master.

Reply:
Solved. The sed is estimated on a low resolution image for a model at high resolution, so the integration is wrong and needs a factor h**2. Now this is replaced by this: 
```
sed *= observation._diff_kernels.image.sum(axis=(-2,-1))
return sed/observation.h ** 2
```
The max of the diff kernel doesn't work because diff kernels are not uni-modal, mootonically decreasing from the center out and might have several pixels at the max.

Reply:
Please use a parameter to choose the normalization method. If the user chooses "max" then use the same normalization as before (since that wasn't acting on the difference kernel, it should still be valid). Until I can verify that this improves initialization on the HSC test dataset (or at least doesn't make it worse) I need to be able to use an initialization that I know works.